catroid/src/main/java/org/catrobat/catroid/content/bricks/PlaceAtBrick.java

```diff
@@ -99,8 +101,18 @@ public void placeVisually() {
 			return;
 		}
 
+		Sprite currentSprite = ProjectManager.getInstance().getCurrentSprite();
+		Formula formulax = getFormulaWithBrickField(BrickField.X_POSITION);
+		Formula formulay = getFormulaWithBrickField(BrickField.Y_POSITION);
 		Intent intent = new Intent(view.getContext(), VisualPlacementActivity.class);
 		intent.putExtra(EXTRA_BRICK_HASH, hashCode());
+		try {
+			intent.putExtra("X", formulax.interpretInteger(currentSprite));
```

Comment:
id prefer to have the interpretInteger portions in a try catch block, saving it into temporary variables and then putting the intent extra outside of the try / catch.